src/Epic/index.stories.tsx

```diff
@@ -0,0 +1,161 @@
+        text: string;
+        baseUrl: string;
+    };
+};
+
+type DataFromKnobs = {
+    heading: string;
+    highlightedText: string;
+    buttonText: string;
+    buttonUrl: string;
+    paragraphs: Array<string>;
+    slotName?: string;
+    componentName?: string;
+};
+
+const Epic: React.FC<EpicProps> = (props) => {
+    const [EpicInner, setEpicInner] = useState<React.FC<EpicProps>>();
+    useEffect(() => {
+        window.guardian = {};
+        window.guardian.automat = {
+            react: React,
+            preact: React,
+            emotionCore,
+            emotionTheming,
+            emotion,
+        };
+        import(/*webpackIgnore: true*/ componentUrl)
+            .then((epicModule: { ContributionsEpic: React.FC<EpicProps> }) => {
+                setEpicInner(() => epicModule.ContributionsEpic);
+            })
+            // eslint-disable-next-line no-console
+            .catch((error) => console.log(`epic - error is: ${error}`));
+    }, []);
+
+    if (EpicInner) {
+        return (
+            <div css={epicWrapper}>
+                {/* eslint-disable-next-line react/jsx-props-no-spreading */}
+                <EpicInner {...props} />
+            </div>
+        );
+    }
+
+    return null;
+};
+
+const buildProps = (data: DataFromKnobs): EpicProps => {
+    return {
+        variant: {
+            heading: data.heading,
+            paragraphs: data.paragraphs,
+            highlightedText: data.highlightedText,
+            cta: {
+                text: data.buttonText,
+                baseUrl: data.buttonUrl,
+            },
+        },
+        tracking: {},
+    };
+};
+
+export const defaultStory = (): ReactElement => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const slotName = text('slotName', 'Epic');
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const componentName = text('componentName', 'Epic');
+    const heading = text('heading', defaultContent.heading);
+    const highlightedText = text('highlightedText', defaultContent.highlightedText);
+    const buttonText = text('buttonText', defaultContent.buttonText);
+    const buttonUrl = text('buttonUrl', defaultContent.buttonUrl);
+    const paragraphs = [];
+
+    // Add existing paragraphs as knobs, and then pad out with empty knobs
+    for (let i = 0; i < TOTAL_PARAGRAPHS; i++) {
+        const name = `paragraph${i + 1}`;
+        const currentParagraph = defaultContent.paragraphs[i];
+        if (currentParagraph) {
+            paragraphs.push(text(name, currentParagraph));
+        } else {
+            paragraphs.push(text(name, ''));
+        }
+    }
+
+    const epicProps = buildProps({
+        heading,
+        highlightedText,
+        buttonText,
+        buttonUrl,
+        paragraphs: paragraphs.filter((p) => p != ''),
+        slotName,
+        componentName,
+    });
+
+    // This is to make the data available to the guPreview add-on:
+    knobsData.set({ ...epicProps });
```

Comment:
Super minor point but this can probably be:

```suggestion
    knobsData.set(epicProps);
```

(I don't think we need to spread/copy `epicProps` here).